base/math.jl

```diff
@@ -124,6 +124,22 @@ for f in (:sinh, :cosh, :tanh, :atan, :asinh, :exp, :erf, :erfc, :expm1)
     @eval ($f)(x::AbstractFloat) = error("not implemented for ", typeof(x))
 end
 
+# functions with special cases for integer arguements
```

Comment:
also a typo "arguements" has an extra e

(add `[ci skip]` to the end of the commit message if a typo is the only thing you change, and run `make check-whitespace` locally)